framework/components/ema-separator/README.md

```diff
@@ -0,0 +1,5 @@
+# Separator
+
+## When to use it
+
+## When not to use it
```

Comment:
There is no need to have the same structure for the readme as ECL components. So feel free to add just a small description of what the component does, and that's it.